tf2onnx/graph.py

```diff
@@ -338,8 +338,20 @@ def __init__(self, nodes, output_shapes=None, dtypes=None, target=None, opset=No
         self.reset_nodes(ops)
 
         # add identity node after each output, in case it is renamed during conversion.
+        nodes_seen = set()
+        multi_output_nodes = set()
         for o in self.outputs:
             n = self.get_node_by_output_in_current_graph(o)
+            if n in nodes_seen:
+                multi_output_nodes.add(n)
+            else:
+                nodes_seen.add(n)
+
+        for o in self.outputs:
+            n = self.get_node_by_output_in_current_graph(o)
+            # TODO: below doesn't work for nodes with multiple outputs. A work around, keep those intact.
```

Comment:
sorry for this regression. We can add the test just we did usually in the yaml file, could you please provide bert model and yaml file?

Reply:
no worry, this is the only model I know that takes multiple outputs from the same node as model output. bert needs a little extra training before its ready to convert ... need to find a place to download it from and its large. Maybe I can add a unit test to simulate the thing.